src/main/java/org/geysermc/discordbot/storage/ServerSettings.java

```diff
@@ -142,6 +142,23 @@ public static boolean shouldNotLogChannel(MessageChannel channel) {
         return dontLog.contains(channel.getId());
     }
 
+    /**
+     * Check if the given channel should be excluded from checking errors
+     *
+     * @param channel The {@link TextChannel} to check
+     * @return If we should exclude the channel
+     */
+    public static boolean shouldNotCheckError(MessageChannel channel) {
+        Guild guild = getGuild(channel);
```

Comment:
We don't need to define this if we just check if it's null, do `getGuild(channel) == null` for example